component/src/main/java/com/celements/structEditor/classes/SelectTagAutocompleteEditorClass.java

```diff
@@ -20,7 +20,8 @@
 
   public static final String SPACE_NAME = "Celements";
   public static final String DOC_NAME = "SelectTagAutocompleteEditorClass";
-  public static final String CLASS_DEF_HINT = SPACE_NAME + "." + DOC_NAME;
+  public static final String CLASS_FN = SPACE_NAME + "." + DOC_NAME;
+  public static final String CLASS_DEF_HINT = CLASS_FN;
 
   public static final ClassField<String> FIELD_AUTOCOMPLETE_SEPARATOR = new StringField.Builder(
       CLASS_DEF_HINT, "select_tag_autocomplete_separator").prettyName("Separator").build();
```

Comment:
please use CLASS_FN for field builders. This can be replaced in the future by a ClassReference